src/status_im/utils/handlers.cljs

```diff
@@ -18,6 +18,12 @@
     (handler db params)
     db))
 
+(defn- pretty-print-event [ctx]
+  (let [[first second & _] (get-coeffect ctx :event)]
```

Comment:
You don't need `& _` while destructuring here.

Reply:
@dmitryn Can you approve? I will make the change

Reply:
@jeluard approved